tests/Propulsion.MessageDb.Integration/Tests.fs

```diff
@@ -18,22 +20,32 @@ module Simple =
         interface TypeShape.UnionContract.IUnionContract
     let codec = FsCodec.SystemTextJson.Codec.Create<Event>()
 
+let createStreamMessage streamName =
+    let cmd = NpgsqlBatchCommand()
+    cmd.CommandText <- "select 1 from write_message(@Id::text, @StreamName, @EventType, @Data, null, null)"
+    cmd.Parameters.AddWithValue("Id", NpgsqlDbType.Uuid, Guid.NewGuid()) |> ignore
+    cmd.Parameters.AddWithValue("StreamName", NpgsqlDbType.Text, streamName) |> ignore
+    cmd.Parameters.AddWithValue("EventType", NpgsqlDbType.Text, "Hello") |> ignore
+    cmd.Parameters.AddWithValue("Data", NpgsqlDbType.Jsonb, """{"name": "world"}""") |> ignore
+    cmd
+
+let writeMessagesToStream (conn: NpgsqlConnection) streamName = task {
+    let batch = conn.CreateBatch()
+    for _ in 1..20 do
```

Comment:
this `20` should be passed from outside so the 20/2000 expected events/handler invocations are forced to be more explicit in the tests